fixtures/soundsation/teste-mobili-array.json

```diff
@@ -0,0 +1,52 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Teste mobili Array",
```

Comment:
I'm not sure what fixture this is.  Soundsation has moving heads called "Teste Mobili", and also an array of moving heads called [AXIS IV MKII](https://www.soundsationmusic.com/en/products/269016-moving-heads/269291-4x32w-rgbw-led-4-head-beam-moving-head), but nothing that matches this name exactly.

In any event, a moving head with just [Intensity, Red, Green] channels is clearly incomplete, and there's no manual or webpage given, so I don't think we can do anything useful with this.